pontoon/settings/base.py

```diff
@@ -585,9 +584,9 @@ def _default_from_email():
     "admin_project": {
         "source_filenames": (
             "js/double_list_selector.js",
+            "js/multiple_item_selector.js",
             "js/multiple_team_selector.js",
```

Comment:
This duplication

Reply:
Now, one is CSS, the other is JS.

Reply:
Apparently my eyes can't see the difference between "item" and "team". Please ignore me ;-)

![cant-see-shit](https://github.com/user-attachments/assets/e7b31681-e399-4afb-af49-9967a2ce867c)